requester.tf

```
@@ -131,7 +131,7 @@ resource "aws_vpc_peering_connection_options" "requester" {
 }
 
 locals {
-  requester_aws_route_table_ids           = distinct(sort(data.aws_route_table.requester.*.route_table_id))
+  requester_aws_route_table_ids           = try(distinct(sort(data.aws_route_table.requester.*.route_table_id)), "")
```

```suggestion
  requester_aws_route_table_ids           = try(distinct(sort(data.aws_route_table.requester.*.route_table_id)), [])
```

although `""` will work as well, let's use the same data type